src/components/CertaintyThresholdGridItem.tsx

```diff
@@ -50,6 +50,8 @@ const CertaintyThresholdGridItem = ({
         width="fit-content"
         borderWidth={1}
         borderColor="transparent"
+        // Classname for spotlighting component in guided tour.
+        className="certainty-threshold"
```

Comment:
Maybe we have constants for these in tours.ts rather than the comments? Should be similarly obvious but less likely to get lost along the way.

Reply:
That makes sense https://github.com/microbit-foundation/ml-trainer/pull/321/commits/4f55791c12c8772556df022b9a4a30ae11fc5eee